runtime/catalog-apicurio/src/test/java/io/aklivity/zilla/runtime/catalog/apicurio/internal/config/ApicurioOptionsConfigAdapterTest.java

```diff
@@ -28,6 +28,8 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import io.aklivity.zilla.runtime.catalog.apicurio.config.ApicurioOptionsConfig;
```

Comment:
This test should also move to the new package, then the import is not required, same as before.